modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,45 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-developer-perspective.adoc
+
+[id="interacting-with-pipelines-using-the-developer-perspective_{context}"]
+= Interacting with Pipelines using the Developer perspective
+
+The *Pipelines* view in the *Developer* perspective lists all the pipelines in a project along with details, such as the namespace in which the pipeline was created, the last pipeline run, the status of the tasks in the pipeline run, the status of the pipeline run, and the time taken for the run.
+
+[Discrete]
+=== Procedure
```

Comment:
=== /.

Reply:
Dont you think it hsoud be the same level as the next topic? Maybe replace === with ==? wdty